database/migrations/2025_04_16_000050_update_jobs_table.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('results', function (Blueprint $table) {
+            $table->unsignedBigInteger('downloaded_bytes')->nullable();
```

Comment:
Can you please add `->after()` method to both of these so they're placed in the table after `download` and `update` respectively? While this only works in MySQL and MariaDB I like to have it for consistency.

Docs: https://laravel.com/docs/12.x/migrations#column-modifiers